Form-Controls/index.html

```diff
@@ -13,15 +13,50 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+         <label for="name">Name</label>
+         <input type="text" id="name" name="name" required>
+         <br>
+         <br>
+         <label for="email">Email</label>
+         <input type="text" id="email" name="name" required>
```

Comment:
Can you see if you can find out if there is a specific input type for email? Right now, if I type my email as 'jennifer', the form will accept this. Usually, an email is like this: username@domain.extension, like jennifer@email.com